src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -155,6 +156,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
                   className={privateChatTabStyles}
                   onClick={selectPrivateChat}
                 >
+                  <FontAwesomeIcon icon={faEnvelope} size="sm" />
```

Comment:
`ButtonNG` supports icons

Reply:
fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54